File: ToDoModel/Email Utilities/CommonWordsModule.vb
Imports System.Text

Public Module CommonWordsModule
    Public Function StripCommonWords(seedString As String, commonWords As IList(Of String)) As String
        Dim input As StringBuilder = New StringBuilder(seedString)
        For Each word As String In commonWords
            input.Replace(word, "")
        Next
        Return input.ToString()
    End Function


End Module

File: ToDoModel/Email Utilities/CommonWordsModule.cs
using System.Collections.Generic;
using System.Text;

namespace ToDoModel
{

    public static class CommonWordsModule
    {
        public static string StripCommonWords(string seedString, IList<string> commonWords)
        {
            var input = new StringBuilder(seedString);
            foreach (string word in commonWords)
                input.Replace(word, "");
            return input.ToString();
        }


    }
}